### ISSUE TITLE
Add FREESP/FREESPT to SvarDOS packages

### ISSUE THREAD

**Issue by evan** [SvarDOS/bugz#33]
This utility https://www.vogons.org/viewtopic.php?f=61&t=78816 fixes slow directory listing operations on old PCs with large FAT partitions, which were once rare but are now common with things like XT-IDE. It would be great to add it to SvarDOS, perhaps even in core.

**Comment by mateuszviste** [comment]
Sounds like a cool tool. I'm not keen on including such things in CORE as I try to keep CORE to the bare minimum, but FREESP would be certainly very welcome in the repo.

Did you check that it works as expected with SvarDOS (ie. with the FreeDOS kernel)?
Any chance you could put the package together? It's basically a matter of putting the tool in some PROGS\FREESP directory and adding a two-line "FREESP.LSM" file in APPINFO\. Hence such directory layout:

APPINFO\FREESP.LSM
PROGS\FREESP\(here all files related to FREESP)

And optionally an unstructured zip file with the FREESP source code, if available.

On the long term it would be probably nice to have such free-space-computation algorithm included inside the FreeDOS kernel, if it truly speeds up things without any side effects.

**Comment by evan** [comment]
Yes, the calculation goes from taking about a minute for a 2GB partition to about 2 seconds on my PC XT. 

It seemed to work fine with both FreeDOS and MS-DOS 6.22. 

I'll try to make a package and test it on my XT.

**Comment by evan** [comment]
This seems to work: https://github.com/evan/freesp.svp/raw/main/freesp-0.4.svp

It doesn't add anything to %DOSDIR%/bin or AUTOEXEC.BAT, but it installs the files.

**Comment by mateuszviste** [comment]
> This seems to work: https://github.com/evan/freesp.svp/raw/main/freesp-0.4.svp

Added to packages, thanks! I have just edited the LSM file to make the description more explicit, and repacked the package with "zip -9rkDX" which made it 13% smaller.
